.github/workflows/plugins-jdk17-test.1.yaml

```diff
@@ -56,6 +56,7 @@ jobs:
       matrix:
         case:
           - spring-6.x-scenario
+          - resteasy-6.x-scenario
```

Comment:
JDK17 is resteasy running minimal requirement?

Reply:
minimal requirement is tomcat 10 and jdk11, should i create a new github workflow for jdk11 tests?

Reply:
But your plugin is still compiled at JDK8 level only, right?

Reply:
this plugin can be compiled at jdk8, and run with resteasy+tomcat in jdk11 (or later)